config/jobs/kubernetes-sigs/sig-windows/release-master-windows.yaml

```diff
@@ -8,6 +8,14 @@ presets:
     value: /etc/ssh-key-secret/ssh-public # from preset-k8s-ssh label
   - name: K8S_SSH_PRIVATE_KEY_PATH
     value: /etc/ssh-key-secret/ssh-private # from preset-k8s-ssh label
+  - name: GINKGO_FOCUS_PARALLEL
+    value: \[Conformance\]|\[NodeConformance\]|\[sig-windows\]|\[sig-apps\].CronJob|\[sig-api-machinery\].ResourceQuota|\[sig-network\].EndpointSlice
+  - name: GINKGO_SKIP_PARALLEL
+    value: \[LinuxOnly\]|\[Serial\]|\[Slow\]|\[alpha\]|GMSA|Guestbook.application.should.create.and.stop.a.working.application|device.plugin.for.Windows
+  - name: GINKGO_FOCUS_SERIAL
+    value: (\[sig-windows\]|\[sig-scheduling\].SchedulerPreemption|\[sig-autoscaling\].\[Feature:HPA\]|\[sig-apps\].CronJob).*(\[Serial\]|\[Slow\])|(\[Serial\]|\[Slow\]).*(\[Conformance\]|\[NodeConformance\])
+  - name: GINKGO_SKIP_SERIAL
+    value: \[LinuxOnly\]|device.plugin.for.Windows
```

Comment:
I like breaking these out so they are not duplicated across the version. Much easier to maintain. did you double check that we didn't have specific tests enable for specific releases?

If we did have a release that needed a test enabled/disabled we could do that on the test it's self?  

For instance it looks like we dropped the skip for 1.20 here: https://github.com/kubernetes/test-infra/pull/20970

Reply:
Yea! big +1 for this change!

Reply:
we can do something like `$(GINKGO_SKIP_SERIAL)|\[sig-scheduling\].SchedulerPredicates.\[Serial\].validates.that.there.is.no.conflict.between.pods.with.same.hostPort` for special cases but in general, most of the jobs follow the same `GINKGO_FOCUS` and `GINKGO_SKIP` across different releases